loglist/findlog/main.go

```diff
@@ -33,8 +33,8 @@ import (
 )
 
 var (
-	logList           = flag.String("log_list", loglist.LogListURL, "Location of master log list (URL or filename)")
-	logListSig        = flag.String("log_list_sig", loglist.LogListSignatureURL, "Location of log list signature (URL or filename)")
+	logList           = flag.String("log_list", loglist.AllLogListURL, "Location of master log list (URL or filename)")
+	logListSig        = flag.String("log_list_sig", "", "Location of log list signature (URL or filename)")
```

Comment:
Not using sigs by default is intentional?

Reply:
Signatures are only checked if a public key is provided (which it isn't, by default).  Changing the `--log_list_sig` flag is just to make the default values match (and the all-logs-list isn't signed).